digits/model/tasks/caffe_train.py

```diff
@@ -1156,14 +1200,25 @@ def can_view_weights(self):
         return False
 
     @override
-    def infer_one(self, data, snapshot_epoch=None, layers=None, gpu=None):
+    def infer_one(self,
```

Comment:
If I can start a useless style discussion for a moment ...

Which do you prefer? I'm not convinced either way.
```py
# fewer lines used
# arguably more natural looking
def hello_world(greeting,
                name):
    print '%s %s!' % (greeting, name)

# fewer total characters uses (less spaces)
# diff is robust to changes in function name
# diff is robust to changes in function arguments
# weird close parens at end
def hello_world(
        greeting,
        name,
):
    print '%s %s!' % (greeting, name)
```
flake8 allows both, which is no help.